types/v3/tech-record/get/trl/complete/index.d.ts

```diff
@@ -352,10 +352,15 @@ export interface TechRecordGETTRLComplete {
   techRecord_authIntoService_datePending?: string | null;
   techRecord_authIntoService_dateAuthorised?: string | null;
   techRecord_authIntoService_dateRejected?: string | null;
+  techRecord_dimensions_axleSpacing?: AxleSpacing[];
```

Comment:
if these arrays are optional do we need them to accept null also? or are they always an array or undefined?

Reply:
yeah sure I can add the null "fields" to the get types, as we discussed earlier

Reply:
Actually no, it can't be null from looking a the current backend validation.

Reply:
![image](https://github.com/dvsa/cvs-type-definitions/assets/104442665/ef384381-1859-475c-b67f-5eb72c44bc8a)